types/tests/bootstrap.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
```

Comment:
ok ,so I assume we'll have one `bootstrap` file that will take care of any proto build across the entire codebase, correct?